packages/app/src/components/Body.tsx

```diff
@@ -18,10 +20,15 @@ export const Body = ({ yOffset }: Props) => {
   if (!data) return null;
 
   return (
-    <Upset
-      data={data}
-      loadAttributes={3}
-      yOffset={yOffset === -1 ? 0 : yOffset}
-    />
+    <div>
+      { Object.entries(data.sets).length === 0 ? 
```

Comment:
`data` is of type `CoreUpsetData`, and there is a property called `setColumns` on it. It's an array of columns that are of type set. Maybe you can just check the length of `setColumns` rather than using `Object.entries`.